### DISCUSSION TITLE
Gravity from Sentinel Kinematic Orbits

### DISCUSSION THREAD

**Discussion by Namesume** [groops-devs/groops#152] [Q&A] [answered]
Dear Developers,

I'm trying to calculate the gravity field by using  program "030groopsProcessing" of "scenario Gravity from kinematic orbits". But I faced some problems in by using Sentinel-1AB data.
In some epoch, 2022-01-01 for example, no matter I set Sentinel-1A or B as input data, the pod data will be empty after processed by "InstrumentReduceSampling" in the first loop, which directly affects the other prgram.
But in some epoch, 2019-03-01 for example, Sentinel-1B is going to have the same problems, but S1A will pass and get the final result successfully.
I've already compared the data of 1A and 1B on 2019-03-01, but I could not find the difference (except the first and the last epoch). And I've already tried to resample the 1B data, which helps to pass "InstrumentReduceSampling" without being emptied and get the final result. But the program will show "WARNING 0. frequency, (idAxis=0) negative factor =....", with the finally result shows "nan".
Would you please tell me why do these problems occur and how should I solve these problem(mainly S1B)?

Like always thank you in advance for your time and help.

With kind regards,
Haonan Shi

**Comment by tmayerguerr** [comment, marked as answer]
Dear Haonan Shi,

The Swarm kinematic orbits are given with a sampling of 1 second, which is too high for gravity field recovery. Therefore the sampling is reduced to 5s. The Sentinel-1 are given with 10s sampling, but unfortunately not always at times divisible by 10s. I suggest to simply omit InstrumentReduceSampling for the Sentinels.

regards
Torsten

**Reply by Namesume** [reply]
Dear Mr.Torsten

Thank you so much for answering my question! I've already replaced the InstrumentReduceSampling with FileConvert, and gotten the final result successfully. Once again, thank you so much for your help!

With kind regards,
Haonan Shi
